ocaml/lib/consistency_test.ml

```diff
@@ -84,30 +84,33 @@ end
 (* deriver *)
 let account_update_of_json, _account_update_to_json =
   let deriver =
-    Account_update.Graphql_repr.deriver @@ Fields_derivers_zkapps.Derivers.o ()
+    lazy
+      ( Account_update.Graphql_repr.deriver
+      @@ Fields_derivers_zkapps.Derivers.o () )
```

Comment:
we compute a bunch of stuff lazily in `consistency_test.ml` and `local_ledger.ml` which was computed at the top level before